api/src/main/resources/extra-mime.types

```diff
@@ -1,4 +1,4 @@
-application/maven+xml         pom
```

Comment:
Not sure where the original came from, but Maven Central is using text/xml so we should probably better stick to that.

Reply:
um.. just have a check here: https://datatracker.ietf.org/doc/html/rfc3023#section-3
And seems rfc also recommends to use text/xml. Ok I'll change it.